src/i18n/i18n.ts

```diff
@@ -7,6 +7,7 @@ const createI18n = (language: string): i18nInstance => {
   const i18n = i18next.createInstance().use(initReactI18next);
 
   i18n.init({
+    compatibilityJSON: 'v3',
```

Comment:
is this an intended change?

Reply:
yes! makes i18n android compatible.

https://stackoverflow.com/questions/70493788/i18nextpluralresolver-your-environment-seems-not-to-be-intl-api-compatible-u

Reply:
got it!!!